### DISCUSSION TITLE
blijvende-zorg

### DISCUSSION THREAD

**Discussion by fjkresin** [fdnd/docs.fdnd.nl#276] [Reflectie]
Voor een paragraaf met het kopje 'blijvende zorg' is het antwoord op 'hoe gaan we daarmee om' wat kort; ik zou dat nog wat verder toelichten.

**Comment by ju5tu5** [comment]
Thanks! Ik heb een zin opgenomen met drie redenen waarom dit ons lukt. Het uitgebreide verhaal staat op aangeven van Liselotte en Iris bij de paragraaf _Aansluiten bij de actualiteit_.
